helm/cluster-operator/templates/deployment.yaml

```diff
@@ -8,6 +8,10 @@ metadata:
     version: {{ .Values.project.version }}
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .Values.project.name }}
+      version: {{ .Values.project.version }}
```

Comment:
This is unrelated considering the PR title. I'd change it to something like `make deployment compatible with K8s 1.16` or something.

Reply:
Title adjusted, hope it's more clear. PTAL

Reply:
Sorry @sslavic didn't notice you changed it and applied my change from the commit, I don't see a way to revert the title, but hope this is OK.

Reply:
All good, your change is more expressive/detailed which I like.

Many ACKs - feel free to ship it!